src/translations/ja/translation.json

```diff
@@ -545,24 +435,154 @@
     "Make bot - moderator, just type /mod streamelements in your chat": "ボット - モデレーターを作成し、あなたのチャットで/mod streamelementsを入力するだけです",
     "Press the \"Add new command\" button": "新しい行動指示を追加する」ボタンを押す",
     "Press \"Activate Command\"": "\"行動指示 \"を押してください",
-    "Player level": "プレイヤーレベル",
-    "Reputation": "親密度",
-    "Commerce": "取引額",
+    "traders-page-description": "Escape from Tarkovのトレーダーについて、必要な情報をすべてご紹介します。ゲーム内で利用できるトレーダーの種類、場所、売却品についてご紹介します。",
     "Trader {{trader}}": "トレーダー{{trader}}",
     "trader-page-description": "Escape from Tarkovに登場するトレーダー{{trader}} の最新情報をお届けします。彼が特定の信頼度で売却するアイテムや、キャッシュバック金銭を最大限に利用して忠誠度を上げる方法について紹介します。",
     "Items with the best cash back prices for leveling": "レベリングに最適なキャッシュバック価格が設定されているアイテム",
     "Spending": "必要取引量",
     "Unlocks at Loyalty Level {{level}}": "信頼度レベルで解錠済み{{level}}",
     "Tasks given by {{traderName}}": "で与えられるタスク{{traderName}}",
-    "traders-page-description": "Escape from Tarkovのトレーダーについて、必要な情報をすべてご紹介します。ゲーム内で利用できるトレーダーの種類、場所、売却品についてご紹介します。",
-    "About Traders": "トレーダーについて",
-    "traders-page-p": "<0>破壊され、包囲されたノルヴィンスクの貿易のバックボーン。Escape from Tarkovでは、各商人は医療品や武器、軍用品など、特定の種類の商品に特化していた。その価格は一般的に高いが、払った分だけ得をする。</0><1>さらに重要なのは、クエストを通じてトレーダーと親密な関係になることで、一般的により良いオファーを受け取れるようになったり、コミッション（販売や購入の際に支払う付加価値）を減らせるなどの利点があることだ。</1><2>さらに、トレーダーは保険や修理（レイド中に死亡した場合に装備品を回収できる）などのサービスも提供しています。</2>",
     "Patch": "パッチ",
     "Wipe start": "ワイプスタート",
     "Wipe end": "ワイプエンド",
     "Ongoing wipe": "オンゴーイングワイプ",
+    "{{count}} days_one": "{{count}}日（単数の場合）",
+    "{{count}} days_few": "{{count}}日（少数の場合）",
+    "{{count}} days_many": "{{count}}日（多数の場合）",
     "{{count}} days_other": "{{count}} 日",
     "Wipe Length": "ワイプの長さ",
     "wipe-length-description": "Escape from Tarkovの平均ワイプ時間に関する最新情報を入手できます。一般的なワイプの長さを知り、次のワイプに備えましょう。",
-    "Average Wipe Length among last 6 wipes:": "直近6回のワイプの平均的な長さ："
-}
+    "Average Wipe Length among last 6 wipes:": "直近6回のワイプの平均的な長さ：",
+    "LL{{level}}": "LL{{level}}",
```

Comment:
The order of keys should be the same as in the original file. I'll run the script that orders them and then we can merge

Reply:
script to correct the key order!

Reply:
I executed a script to fix the order of keys in each JSON file!